### PR TITLE
Update `validate.bat` to fix typo

### LINKED ISSUE #1054: `validate.bat` no executes correctly on windows
### Checked for duplicates

No - I haven't checked

### 🐛 Describe the bug

When I tried to run on Windows, I get this:

image (2)

### 🕵️ Expected behavior

I expected validate to execute

### 📜 To Reproduce

See bug description

### 🖥 Environment Info

Windows OS

### 📚 Version of Software Used

3.6.0

### 🩺 Test Data / Additional context

_No response_

### 🦄 Related requirements

_No response_

### ⚙️ Engineering Details

_No response_

### 🎉 Integration & Test

_No response_

### PR DESCRIPTION
**Pull request by jordanpadams**
## 🗒️ Summary
Updated `project.validate` to `project.version`

Quotes also added to hopefully help with future debugging.

## ⚙️ Test Data and/or Report
Screenshot 2024-11-06 at 9 20 19 AM

## ♻️ Related Issues

Resolves #1054

### CODE REVIEW

File: src/main/resources/bin/validate-refs.bat [hunk 1]
```diff
@@ -51,7 +51,7 @@ set LIB_DIR=%PARENT_DIR%\lib
 if exist "%LIB_DIR%\validate-@project.version@.jar" (
 set VALIDATE_JAR=%LIB_DIR%\validate-@project.version@.jar
 ) else (
-echo Cannot find VTool jar file (validate-@project.version@.jar) in %LIB_DIR%
+echo "Cannot find VTool jar file (validate-@project.version@.jar) in %LIB_DIR%""
```
Comment (nutjob4life) [inline src/main/resources/bin/validate-refs.bat]: Is the double `""` at the end of this line correct?

(And is the whole file using DOS line endings? I see a `^M` here, but just here, but that may be a GitHub display artifact.)

**Review by al-niessner — APPROVED** [review]

**Review by nutjob4life — CHANGES_REQUESTED** [review]
My Windows system is currently offline so this is a visual inspection only—sorry!

File: src/main/resources/bin/validate-refs.bat (hunk 1 above)
Comment (jordanpadams) [reply src/main/resources/bin/validate-refs.bat]: @nutjob4life fixed. not sure about the carriage return? I don't feel that I am ever successful in seeing these.

Comment (nutjob4life) [reply src/main/resources/bin/validate-refs.bat]: @jordanpadams you and me both 😬

**Review by nutjob4life — APPROVED** [review]
👍

**Pull request merged**